src/app/public/modules/description-list/description-list.component.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ContentChildren,
+  ElementRef,
+  HostListener,
+  Input,
+  OnDestroy,
+  QueryList,
+  ViewChild
+} from '@angular/core';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  SkyDescriptionListMode
+} from './types/description-list-mode';
+
+import {
+  SkyDescriptionListOrientation
+} from './types/description-list-orientation';
+
+import {
+  SkyDescriptionListAdapterService
+} from './description-list-adapter-service';
+
+import {
+  SkyDescriptionListContentComponent
+} from './description-list-content.component';
+
+import {
+  SkyDescriptionListService
+} from './description-list.service';
+
+/**
+ * Creates a description list to display term-description pairs.
+ */
+@Component({
+  selector: 'sky-description-list',
+  templateUrl: './description-list.component.html',
+  styleUrls: ['./description-list.component.scss'],
+  providers: [SkyDescriptionListService],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyDescriptionListComponent implements AfterContentInit, OnDestroy {
+
+/**
+ * Specifies a default description to display when no description is provided
+ * for a term-description pair.
+ * @default 'None found'
+ */
+  @Input()
+  public set defaultDescription(value: string) {
+    this.descriptionListService.defaultDescription.next(value);
+  }
+
+  /**
+   * Specifies the width of the list item when `mode` is set to `standard`. By default,
+   * the list items will have responsive widths at different sizes.
```

Comment:
```suggestion
   * the width is responsive to different screen sizes.
```

Reply:
@Blackbaud-AlexKingman, just checking back on this change. I think this wording is clearer than what we have currently, but we can dump the suggestion if you don't think it works. Just let me know.